documentation/book/con-custom-resources-example.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-example-{context}']
+= {Productname} custom resource example
+Each {Productname}-specific custom resource is defined within the specification of a CRD.
```

Comment:
I don't really know what "defined within the specification of" means. I can guess what you mean, but I don't think it's well-known terminology.

```suggestion
Each {Productname}-specific custom resource is conforms to the schema defined by the CRD for the resource's `kind`.
```

Reply:
Changed to:
Each {Productname}-specific custom resource conforms to the schema defined by the CRD for the resource's `kind`.